src/modules/signUp/signUp.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import {  Form,FormGroup,Button,Col,ControlLabel,FormControl,Row,Alert} from 'react-bootstrap';
+import './signUp.scss';
+import Header from '../header/header';
+import AlertMsg from '../alert/alert';
+import * as action from '../../redux/signUp/action';
+
+
+class Signup extends React.Component{
+  constructor(props){
+    super(props);
+    this.state={
+      username:'',
+      password:'',
+      option:'admin'
+    }
+  };
+  onFormSubmit(e){
+    e.preventDefault();
+    var data1={
+      'username':this.state.username,
+      'password':this.state.password,
+      'option':this.state.option
+        }
+     this.props.onSignUp(data1);
+    }
+
+  handleUsernameChange = (e) => {
+    this.setState({username: e.target.value});
+  }
+
+  handlePasswordChange = (e) => {
+    this.setState({password: e.target.value});
+  }
+
+  handleOptionChange=(e)=>{
+   this.setState({option:e.target.value})
+ }
+
+  componentWillReceiveProps(props){
+    if(props.signUp.error === 1){
+       alert('Account already exist')
+    }
+    else if(props.signUp.error===0){
+      this.props.router.push('/login');
+      alert("you are sucessfull sign up");   
```

Comment:
this is a normal alert ?? change it to your generic alerts

Reply:
ok sir